feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt

```diff
@@ -0,0 +1,169 @@
+package com.iteneum.profile.presentation
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.LocationOn
+import androidx.compose.material.icons.outlined.Mail
+import androidx.compose.material.icons.outlined.Phone
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import com.iteneum.designsystem.components.LpEditFloatingActionButton
+import com.iteneum.designsystem.components.LpOutlinedButton
+import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.profile.R
+
+@Composable
+fun ProfileView(
+
```

Comment:
If this function doesn't have any parameter please delete the space and add documentation to this function

Reply:
Hi Melissa, I'm agree, so I put a TODO comment, because I do not know how some parameters will be declare, such as navigation or the view model, this with the purpose to announce there will be those parameters